servers/audio/effects/audio_effect_reverb.cpp

```diff
@@ -38,7 +38,7 @@ void AudioEffectReverbInstance::process(const AudioFrame *p_src_frames, AudioFra
 		r.set_predelay_feedback(base->predelay_fb);
 		r.set_highpass(base->hpf);
 		r.set_room_size(base->room_size);
-		r.set_damp(base->damping);
+		r.set_damp(base->reflectivity);
```

Comment:
If we do this change, `set_damp` should likely also be changed, i.e. the `Reverb` class needs to be adjusted too.

Reply:
I'll adjust the `Reverb` class for now.